kubernetes/apps/charts/jupyterhub/values.yaml

```diff
@@ -8,7 +8,7 @@ jupyterhub:
     defaultUrl: "/lab"
     image:
       name: ghcr.io/cal-itp/calitp-py
-      tag: hub-v10
+      tag: hub-v11
```

Comment:
@atvaccaro is this ok to go into production?

Reply:
yes! thanks